src/lib/objects.js

```diff
@@ -0,0 +1,84 @@
+const createPerson = (name, age) => {
```

Comment:
As before you should run the formatter over this, theres a few indentation issues like here and some spaces missing in a few places. Would be much easier to do this and have it do the heavy lifting for you rather than go through it all manually